apps/code-examples/src/app/code-examples/autonumeric/autonumeric/currency/autonumeric-demo.component.html

```diff
@@ -1,6 +1,6 @@
 <form [formGroup]="formGroup" novalidate>
   <sky-input-box>
-    <label class="sky-control-label" [for]="autonumericInput.id">
+    <label class="sky-font-data-label" [for]="autonumericInput.id">
```

Comment:
Think we can leave these as `sky-control-label` based on the way input box is built, but would need to verify

Reply:
I can revert these, but _should_ the input box component work with `.sky-font-data-label` as well as `.sky-control-label`?